lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
 <label class="u-field-title" aria-hidden="true">
     <%- title %>
 </label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<span class="sr" for="u-field-value-<%- id %>"><%- screenReaderTitle %></span>
```

Comment:
If this code is read-only specific, then it would be great if we could make this an ```<h2>```.  The Heading would include both the screen reader text and the username.